package.json

```diff
@@ -113,6 +113,7 @@
     "http-errors": "^1.8.0",
     "jquery": "^3.6.0",
     "jwt-decode": "^3.1.2",
+    "lodash": "^4.17.21",
```

Comment:
Adding `lodash` to the project. It's a useful library with lots of utilities. I'm using it to access object variables using dot-notation as a string. See below where I specified `'licence.conditionalReleaseDate'`